nupic/research/frameworks/pytorch/imagenet/imagenet_experiment.py

```diff
@@ -653,6 +659,11 @@ def aggregate_validation_results(cls, results):
         result.update(aggregate_eval_results(results))
         return result
 
+    @classmethod
+    def aggregate_pre_experiment_results(cls, results):
+        if all(results):
```

Comment:
Is this something that only happens on `pre_experiment`?
Maybe this check should be placed in `aggregate_validation_results` instead to be safe.

Reply:
That's a fair question. The pre_experiment is optionally run (e.g. `-1 in epochs_to_validate`) while validate should return a dict, or at least that's the convention in the code right now.